pylearn2/models/autoencoder.py

```diff
@@ -772,6 +772,46 @@ def _modify_updates(self, updates):
             autoencoder.modify_updates(updates)
 
 
+class StackedDenoisingAutoencoder(DeepComposedAutoencoder):
+    """
+    A stacked denoising autoencoder learns a representation of the input by
+    reconstructing a noisy version of it.
+
+    Parameters
+    ----------
+    autoencoders : list
+        A list of autoencoder objects.
+    corruptor : object
+        Instance of a corruptor object to use for corrupting the
+        input.
+    """
+    def __init__(self, autoencoders, corruptor):
+        super(StackedDenoisingAutoencoder, self).__init__(autoencoders)
+        self.corruptor = corruptor
+
+    def reconstruct(self, inputs):
```

Comment:
You could just wrap `AbstractAutoencoder.reconstruct` instead of duplicating the docstring.